src/components/home/components/Footer.js

```diff
@@ -1,31 +1,28 @@
 import React from 'react';
-import { Col, Layout, Row } from 'antd';
+import { Layout } from 'antd';
 
-import { FOOTER_STRINGS } from './constants';
-import { components } from '../../productsAvailable';
+import { FOOTER_DATA, FOOTER_STRINGS } from './constants';
 
-const { ProductsAvailable } = components;
-const { Content, Footer } = Layout;
-const { CENTER, FLEX, FOOTER, FOOTER_CONTENT, FOOTER_TEXT, LOWER_FOOTER } = FOOTER_STRINGS;
+const { Footer } = Layout;
+const { CONTAINER, DYNAMIC_FOOTER, ROW } = FOOTER_STRINGS;
 
-//This contains the productsavailable component
-class  AppFoot extends React.Component {
+const footers = FOOTER_DATA.map(footer => (
```

Comment:
coding is becoming more interesting for me

Reply:
Yeah, maps are some of the most simple and powerful tools in software development

Reply:
Wow
We are getting there
footer really looks good